### ISSUE TITLE
GetEntityByReferenceId documentation

### ISSUE THREAD

**Issue by dahifi** [officecenter/Autotask#44]
Readme says to use this but it's not present as a method. GetReferenceEntityByID and get ExternalEntityByThisEntityID don't seem to work for me either. 

```
$contract = Get-AtwsContract -id 29683090 -GetReferenceEntityById AccountID
Get-AtwsData : Cannot bind argument to parameter 'Entity' because it is an empty string.
At C:\Users\MichaelWade\Documents\WindowsPowershell\Cache\501990\Dynamic\Get-AtwsContract.ps1:622 char:40
+ ...     $Result = Get-Atwsdata -Entity $Field.ReferenceEntityType -Filter ...
+                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-AtwsData], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,Get-AtwsData
```

**Comment by klemmestad** [comment]
That is  a regression. The fix is embarrasingly simple, but I need to rebuild the module to fix it. Have another fix in the pipeline, too, but no major changes. Should be able to release quite quickly.

**Comment by dahifi** [comment]
I'm keeping you on your toes, aint I?

**Comment by klemmestad** [comment]
Yep. Although I see that this automatic references does not take code branches into account. The code is coming to the master branch now. That should trigger a new release of the production module with the changes from Development.

**Comment by klemmestad** [comment]
Should be fixed in 1.6.2.17 (out now)
